css/webots-doc.css

```diff
@@ -253,3 +253,162 @@
   font-family: monospace;
   width: 80%;
 }
+
+.blog {
+  background-color: #ede9e8;
+  background-image: url(https://www.cyberbotics.com/files/repository/www/wwi/8.5/crossword.png);
```

Comment:
For the moment, whatever the branch, the cyberbotics site loads resources (webots-doc.css and viewer.js) from the 8.5 folder, so for consistency, i introduced this file here. I can move it if you wish.